sdk/cosmos/azure-cosmos-spark_3_2-12/docs/metrics.md

```diff
@@ -1,60 +0,0 @@
-# Cosmos DB client-side metrics
```

Comment:
naive question: is this some sort of deprecation action?

Reply:
Yes - I birefly touched on it in the PR description. The prvious model was so complicated that I am not aware of any customer using it. The new model provides more fucntionality easier - so, I think it is ok given the numbe rof customers who use this model would be extremely small (if any)